sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/TestUtils.java

```diff
@@ -96,7 +146,11 @@ public static String getConnectionString(boolean withSas) {
      * @return The fully qualified domain name for the service bus resource.
      */
     public static String getFullyQualifiedDomainName() {
-        return System.getenv("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
+        return getPropertyValue("AZURE_SERVICEBUS_FULLY_QUALIFIED_DOMAIN_NAME");
+    }
+
+    public static String getEndPoint() {
```

Comment:
rename to: getEndpoint()

Reply:
It has been verified that this getter method can be replaced by `TestUtils.getEndPoint()`, now fixed in the new version.

Reply:
Looks like this change regressed

Reply:
Sorry for missing this fix in the previous revision, now fixed in the new version.